tests/queries/test_explain.py

```diff
@@ -57,6 +57,8 @@ def test_postgres_options(self):
             test_options.append({'summary': True})
         if connection.features.is_postgresql_12:
             test_options.append({'settings': True})
+        if connection.features.is_postgresql_13:
+            test_options.append({'wal': True})
```

Comment:
We need to add `analyze: True`, currently it fails: `django.db.utils.DataError: EXPLAIN option WAL requires ANALYZE`.

Reply:
Mea culpa. Will fix.